### DISCUSSION TITLE
关于解锁了主题但是领不了的问题

### DISCUSSION THREAD

**Discussion by wuwenjun9939** [Meloong-Git/PCL#3919] [提问] [answered]
如题，我系统经历了一次重装之后，之前的识别码发生了改变，如果是这个问题的话，我还能再领一次隐藏的主题吗？或者说，还有机会领吗？还是说要再上传个bug？

**Comment by wuliaodexiaoluo** [comment, marked as answer]
你可以在原来的 issue 上再次要求解锁，解锁完成后请记得在帮助库上备份 pcl 设置，这份注册表在换计算机前将始终保持有效

**Reply by wuwenjun9939** [reply]
谢了（我也不会换电脑）
